backend/LexBoxApi/Services/FwLiteReleases/FwLiteReleaseService.cs

```diff
@@ -96,4 +100,43 @@ public async ValueTask InvalidateReleaseCache()
         activity?.AddTag(FwLiteReleaseVersionTag, null);
         return null;
     }
+
+    public async ValueTask<string> GenerateAppInstaller(CancellationToken token = default)
+    {
+        var windowsRelease = await GetLatestRelease(FwLiteEdition.Windows, token);
+        if (windowsRelease is null) throw new InvalidOperationException("Windows release not found");
+        var version = ConvertVersionToAppInstallerVersion(windowsRelease.Version);
+        //lang=xml
+        return $"""
+<?xml version="1.0" encoding="utf-8"?>
+<AppInstaller
+ Uri="https://lexbox.org/api/fwlite-release/download-latest?edition=windowsAppInstaller"
+ Version="{version}"
+ xmlns="http://schemas.microsoft.com/appx/appinstaller/2021">
+ <MainBundle
+   Name="FwLiteDesktop"
+   Publisher="CN=&quot;Summer Institute of Linguistics, Inc.&quot;, O=&quot;Summer Institute of Linguistics, Inc.&quot;, L=Dallas, S=Texas, C=US"
```

Comment:
Probably this? Or...where did you get this text?
```suggestion
   Publisher="CN=&quot;SIL Global&quot;, O=&quot;SIL Global&quot;, L=Dallas, S=Texas, C=US"
```

Reply:
Nope, this must match the Common Name in our windows signing certificate, if you notice it's the same value used in the msix config file for Maui.